src/escenas/libroPrimaria/EscenaDesdeMapa.ts

```diff
@@ -440,6 +456,22 @@ class GeneradorDeCasillaMaybe implements GeneradorDeCasilla {
         return 0 < proba && proba < 1;     
     }
 }
+//TODO: se asume que siempre es un simple
+class GeneradorDeCasillaAnd implements GeneradorDeCasilla {
+    generador2: GeneradorDeCasilla
+
+    constructor(private generador1 : GeneradorDeCasilla, generador2: GeneradorDeCasilla) {this.generador2 = generador2}
```

Comment:
es lo mismo que esto? no conocía este feature de ts!!
```suggestion
    constructor(private generador1 : GeneradorDeCasilla, private generador2: GeneradorDeCasilla) {}
```